trunk/includes/class-simple-staff-list-activator.php

```diff
@@ -164,7 +164,7 @@ public static function activate( $is_forced ) {
 			update_option( '_staff_listing_custom_name_plural', $default_name_plural );
 		}
 
-		if ( 'forced activation' != $is_forced ) {
+		if ( true == $is_forced ) {
```

Comment:
Thanks for this. Change 167 to `if ( true === $is_forced ) {` and this I'll get it merged in.